devenv/dev-dashboards/panel_tests_graph_time_regions.json

```diff
@@ -44,6 +44,7 @@
       "linewidth": 2,
       "links": [],
       "nullPointMode": "null",
+      "paceLength": 10,
```

Comment:
what is paceLength?

Reply:
think it should be spaceLength

Reply:
Have no idea where that came from :smile: where is spaceLength used and what for?

Reply:
Fixed by #16205